k8s-install/canal-kubernetes-datastore.yaml

```diff
@@ -71,31 +72,41 @@ spec:
         scheduler.alpha.kubernetes.io/critical-pod: ''
         scheduler.alpha.kubernetes.io/tolerations: |
           [{"key": "dedicated", "value": "master", "effect": "NoSchedule" },
-           {"key":"CriticalAddonsOnly", "operator":"Exists"}]
+           {"key": "CriticalAddonsOnly", "operator": "Exists"}]
```

Comment:
These will be respected by Kubernetes < v1.6, but v1.6+ use the `tolerations` field.

We may need a v1.6 version of this manifest as well.  Check out how it's done in the kubeadm section.